platform/src/main/java/org/stellar/anchor/platform/controller/Sep10Controller.java

```diff
@@ -88,6 +90,15 @@ public SepExceptionResponse handleSepValidationException(Exception ex) {
     return new SepExceptionResponse(ex.getMessage());
   }
 
+  @ExceptionHandler({
+    SepNotAuthorizedException.class,
+  })
+  @ResponseStatus(value = HttpStatus.FORBIDDEN)
+  public SepAuthorizationExceptionResponse handleSepAuthorizationException(Exception ex) {
+    errorEx(ex);
+    return new SepAuthorizationExceptionResponse(ex.getMessage());
+  }
```

Comment:
This piece of code seems redundant with what we already had in the Global error handler:

https://github.com/stellar/java-stellar-anchor-sdk/blob/8cd8d07a4ff8ce202158c4c064cde542bbc4acc2/platform/src/main/java/org/stellar/anchor/platform/controller/GlobalControllerExceptionHandler.java#L33-L38

Reply:
i thought we agreed to return 403 errors for those instead of a generic 400?

Reply:
Oh, my mistake. I highlighted the wrong piece of code in my comment. Here is what I should have highlighted: 

https://github.com/stellar/java-stellar-anchor-sdk/blob/8cd8d07a4ff8ce202158c4c064cde542bbc4acc2/platform/src/main/java/org/stellar/anchor/platform/controller/GlobalControllerExceptionHandler.java#L40-L45

If you look at the Global exception handler ☝️ , it does the same thing as the structure you added:
1. Intercept `SepNotAuthorizedException`.
2. Use `HttpStatus.FORBIDDEN` in the response status code.
3. The response body will be the same: 
  ```json
  {
    "error": "{originalExceptionMessage}"
  }
  ```

Reply:
So my point is that you added a new exception handler for `SepNotAuthorizedException` in Sep10Controller that overrides the default exception handler in GlobalControllerExceptionHandler and does the exact same thing as the default exception handler, so you're basically adding some code that's not necessary.